src/workerd/io/io-channels.h

```diff
@@ -4,6 +4,9 @@
 
 #pragma once
 
+#include "workerd/io/actor-cache.h"
+#include "workerd/io/io-own.h"
```

Comment:
Nit: are these includes actually needed in io-channels.h or is this just to make the decls available in the other files changed in this PR? If the latter, I'd prefer if the includes were added where they are actually used.

Reply:
Had initially needed these includes, but took the PR in another direction. Thanks for catching!